dot/state/block.go

```diff
@@ -651,11 +652,25 @@ func (bs *BlockState) GetRuntime(hash *common.Hash) (runtime.Instance, error) {
 	return bs.bt.GetBlockRuntime(*hash)
 }
 
+//  GetBestBlockRuntime returns the current runtime instance for the best block
+func (bs *BlockState) GetBestBlockRuntime() (instance runtime.Instance) {
+	instance, err := bs.bt.GetBlockRuntime(bs.BestBlockHash())
+	if err != nil {
+		logger.Errorf("error retrieving block runtime: %w", err)
+	}
+	return
```

Comment:
So.... on that one....

I guess this is taking it from memory and it should ALWAYS be there right? If so replace the error log with a panic.
We definitely don't want this to go unnoticed in an error log.

```suggestion
	instance, err := bs.bt.GetBlockRuntime(bs.BestBlockHash())
	if err != nil {
		panic(fmt.Sprintf("we should always succeed getting the best block runtime but an error occurred: %s", err))
	}
	return instance
```